src/cascadia/TerminalSettingsModel/CascadiaSettings.cpp

```diff
@@ -15,6 +15,7 @@
 #include <shellapi.h>
 #include <shlwapi.h>
 #include <til/latch.h>
+#include <til/env.h>

```

Comment:
are we using anything from env here? the `details` bit? we may want to move that to til/string and move it out of details

(by policy, one shouldn't use `x::details` outside of `x` :))